### DISCUSSION TITLE
播客托管

### DISCUSSION THREAD

**Discussion by sfyumi** [halo-dev/halo#6637] [Ideas]
### 你当前使用的版本

1.4.12

### 描述一下此特性

现在播客越来越流行了，但是没有一款好的开源的播客托管工具，希望 halo 能支持播客托管功能。

### 附加信息

可以参考一下《疯投圈》的播客页面 https://crazy.capital/

**Comment by ruibaby** [comment]
目前不考虑在内部集成这个功能，后续可以考虑使用插件的形式接入。

**Comment by LinuxSuRen** [comment]
播客确实有变得流行的趋势，可以考虑先支持添加RSS的支持

**Comment by JohnNiang** [comment]
@LinuxSuRen Halo 一直是支持 RSS 的，请看：https://halo.run/rss.xml 和 https://halo.run/atom.xml 。

**Comment by LinuxSuRen** [comment]
可以通过添加 RSS 来收听播客？

**Comment by ruibaby** [comment]
@JohnNiang 播客也是有订阅的。比如：

https://moon.fm/podcasts/710343/%F0%9F%8E%99%E8%B0%90%E6%98%9F%E8%81%8A%E5%A4%A9%E4%BC%9A
https://feeds.danlirencomedy.com/xxlth.xml

**Comment by mqllin** [comment]
大家在什么时候会听播客？我只在iphone的ios系统中见过系统自带的播客APP，但我从来没有点开过。喜马拉雅之类的APP也算播客吗？

**Comment by LinuxSuRen** [comment]
开车、坐地铁（公交）、工作（coding）的时候都可以听。喜马拉雅算是一个综合性的播客，还有小宇宙等。另外，欢迎收听《开源面对面》里分享的开源相关的故事、工作岗位等。

https://www.ximalaya.com/album/53320813

**Comment by LinuxSuRen** [comment]
抱歉打扰各位（我从 contributor 列表中找到前 4 位）：

@ruibaby @JohnNiang @guqing @wtune 

我代表[《开源面对面》](https://github.com/opensource-f2f/episode/blob/master/open-source-projects.md)播客制作团队，想邀请 halo 项目的活跃贡献者聊聊这个项目的故事。

> 另外，除了上面我提到的贡献者外，任何该项目的贡献者以及用户都可以参与。

**Comment by ruibaby** [comment]
@LinuxSuRen 可以，非常荣幸。

**Comment by guqing** [comment]
> @LinuxSuRen 可以，非常荣幸。

非常感谢 @LinuxSuRen 的邀请，将由 @ruibaby 作为 halo 代表参与到此次活动中来，如果需要其他成员配合，我们也十分荣幸🎉🎉

**Comment by LinuxSuRen** [comment]
我希望能邀请三维左右的共建者（contributor）来一起聊，大家先抽空听听《开源面对面》之前的节目，熟悉下录制的风格。我们本周尽量能抽个时间来做个前期交流（pre-talk），可以用 https://doodle.com/  这个工具来对交流时间做下投票！非常期待！

**Comment by LinuxSuRen** [comment]
hi @sfyumi ，我做了一个开源播客平台的 POC，欢迎体验、共建 http://103.61.38.146:30001/
